### DISCUSSION TITLE
How I can open an external File with the specific Programm

### DISCUSSION THREAD

**Discussion by beanoccio** [artisan-roaster-scope/artisan#1731] [General]
I want to open an MSAccess database file (.accdb) with MSaccess.exe.
What should the command look like?

![image](https://github.com/user-attachments/assets/69ea3456-c1df-4277-93ae-f81130cdec58)

**Comment by roasterdave** [comment]
That looks correct.  A similar command with Excel works for me.  What happens when you click the DB button?

**Reply by beanoccio** [reply]
Now I've tried it with an Excel file and it works perfectly.

If I try the same thing with Access, only an instance of Access opens in the task manager, but not the frontend database what is the target.

I think there's a problem with MSAccess. I have to research this in the MS Access community.
I've also tried it with a VBScript. It works with a double click.

DB.vbs
Option Explicit
Dim Access
Set Access = CreateObject("Access.Application")
Access.OpenCurrentDatabase("D:\Beanocco\Beanoccio-DB\Beanoccio DB.accdb")
Access.Visible = True
MsgBox "Here I am!"

But I can't start this script with Artisan's "external program" function.

![image](https://github.com/user-attachments/assets/ef8a572c-5519-449b-8da9-a95b5ab9e1e0)

**Comment by roasterdave** [comment]
You might try a simple batch script that has the one line below.  You can test the batch script outside of Artisan.  Then, assuming Access starts properly, call the batch file from Artisan.

![image](https://github.com/user-attachments/assets/dda018a9-71ce-402f-9eea-fbf1d4c6334c)

**Reply by beanoccio** [reply]
I have now tested calling the Access database on another PC. There it works without any problems with the direct call in (Events-> Call program), even without going through "vbs" or "bat" files.

There is obviously a problem with my Windows or Access installation. Thank you for the productive conversation.

**Comment by beanoccio** [comment]
That's the strange thing. A simple batch script doesn't work. of course with Excel but not with access database.
That's why I used a vbscript. I think this is a Microsoft bug or something.
I'm going to do a fresh installation of Access.
